packages/swimlane/data_stream/audit_logs/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,199 @@
+---
+description: Pipeline for Swimlane Audit Logs
+processors:
+  - set:
+      field: event.original
+      copy_from: message
+  - json:
+      field: message
+      target_field: json
+      ignore_failure: true
+  - drop:
+      if: ctx.json == null || !(ctx.json instanceof Map)
+  - remove:
+      field: 
+        - message
+        - input
+        - log
+      ignore_missing: true
+  - rename:
+      field: json
+      target_field: swimlane.audit_log
+      ignore_missing: true
+  - drop:
+      if: ctx.swimlane?.audit_log.description.toLowerCase().contains('read') || ctx.swimlane.audit_log.description.toLowerCase().contains('created a new record')
```

```suggestion
      if: >-
        ctx.swimlane?.audit_log?.description != null &&
        (
          ctx.swimlane.audit_log.description.toLowerCase().contains('read') ||
          ctx.swimlane.audit_log.description.toLowerCase().contains('created a new record')
        )
```
The current code will error with a null deref in the case that `ctx.swimlane?.audit_log?.description` is null or if  `ctx.swimlane?.audit_log` is null.

---

Fixed in https://github.com/elastic/integrations/pull/13972/commits/815c09309237d43e51cdd39fc29bdb42561fccf9